doc/reference/power_management/index.rst

```diff
@@ -339,7 +339,7 @@ Device Set Power State
 
 .. code-block:: c
 
-   int device_set_power_state(struct device *device, u32_t device_power_state);
+   int device_set_power_state(struct device *device, u32_t device_power_state, void *arg, device_pm_cb cb);
```

Comment:
I suggest swapping cb and arg arguments. This would result in more convenient order.

Reply:
do you want to send the arg to callback() from driver set power function?

Reply:
Yes, This should be opaque value which is passed from device_set_power_state() caller to callback.

Reply:
Ok